lang/lsp/src/server.rs

```diff
@@ -30,9 +32,11 @@ impl LanguageServer for Server {
     async fn initialize(&self, params: InitializeParams) -> jsonrpc::Result<InitializeResult> {
         let capabilities = capabilities();
         #[cfg(not(target_arch = "wasm32"))]
+        // FIXME: Use `workspace_folders` instead of `root_uri`.
```

Comment:
You probably know what this comment means :) Can you add a sentence for me so that I know what hack we are currently using here?